src/components/Button/Button.tsx

```diff
@@ -30,18 +29,19 @@ export class Button extends React.Component<IButtonProps, IButtonState> {
   }
 
   public render(): JSX.Element {
-    let { buttonType, children, icon, description, ariaLabel, ariaDescription, elementType, disabled } = this.props;
+    let { buttonType, children, icon, description, ariaLabel, ariaDescription, href, disabled } = this.props;
     let { labelId, descriptionId, ariaDescriptionId } = this.state;
 
-    const tag = this.props.elementType === ElementType.button ? 'button' : 'a';
+    const renderAsAnchor: boolean = !!href;
+    const tag = !renderAsAnchor ? 'button' : 'a';
 
-    const className = css(this.props.className, 'ms-Button', {
+    const className = css((this.props.className), 'ms-Button', {
       'ms-Button--primary': buttonType === ButtonType.primary,
       'ms-Button--hero': buttonType === ButtonType.hero,
       'ms-Button--compound': buttonType === ButtonType.compound,
       'ms-Button--command': buttonType === ButtonType.command,
       'ms-Button--icon': buttonType === ButtonType.icon,
-      'disabled': (elementType === ElementType.anchor && disabled)
+      'disabled': (!renderAsAnchor && disabled)
```

Comment:
I didn't even realize it because I wrote this reasoning about what it should be and not what was there. What was there appears to be buggy in the first place 😦